client.go

```diff
@@ -18,11 +18,11 @@ func getRedisConfig(cfgFile string) *RedisConfig {
 	return &config
 }
 
-func newRedisClient(cfgFile string) *redis.Client {
+func newRedisClient(cfgFile string, Addr string) *redis.Client {
```

Comment:
[optional] can make an interface for this as we will have many options.

Reply:
it is already there. RedisConfig